lib/processors/themeBuilderWorker.js

```diff
@@ -0,0 +1,88 @@
+import workerpool from "workerpool";
+import themeBuilder from "./themeBuilder.js";
+import fsInterface from "@ui5/fs/fsInterface";
+import {createAdapter, createWriterCollection, createResource} from "@ui5/fs/resourceFactory";
+
+/**
+ * Task to build library themes.
+ *
+ * @private
+ * @function default
+ * @static
+ *
+ * @param {object} parameters Parameters
+ * @param {object} parameters.allResources Array of Uint8Array transferable objects
+ * 	with all the resources needed to build the css files. By nature those are @ui5/fs/Resource.
+ * @param {object} parameters.themeResources Input array of Uint8Array transferable objects
+ * 	that are the less sources to build upon. By nature those are @ui5/fs/Resource.
+ * @param {object} parameters.options Less compiler options
+ * @returns {Promise<Uint8Array[]>}
```

Comment:
Type differs from type described for `makeTransferableResource`.

Reply:
Thanks

Reply:
Done